src/wirecloud/platform/workspace/routes.py

```diff
@@ -33,6 +33,7 @@
 from src.wirecloud.commons.utils.wgt import WgtFile
 from src.wirecloud.platform.markets.utils import get_local_catalogue
 from src.wirecloud.platform.preferences.crud import update_workspace_preferences
+from src.wirecloud.platform.search import add_workspace_to_index, update_workspace_in_index
```

Comment:
No usas `update_workspace_in_index`.

```suggestion
from src.wirecloud.platform.search import add_workspace_to_index
```